_test/test_data_loaders/test_loader_utilites.m

```diff
@@ -44,25 +44,25 @@ function test_correct_Folder_found(this)
        function test_correct_DSinCorrectFolder_found(this)
             [DS_info,ds_path] = find_dataset_info(f_name(this,'MAP11014.nxspe'),'data','data'); 
             
-            assertEqual(DS_info.Dims,[30,28160]);
+            assertEqual(DS_info.Dataspace.Size,[30,28160]);
```

Comment:
where did this API change come from?

Reply:
hdf5info , which Matlab claims will be removed soon and h5info which is current (and is here for a while) return different types of structures. I took this ocasion to make the change.

To be fair, h5info returns more sensible outputs, so It looks like for the better